client/my-sites/customer-home/cards/features/site-preview/index.tsx

```diff
@@ -49,9 +49,15 @@ const ThumbnailWrapper = ( { showEditSite, editSiteURL, children }: ThumbnailWra
 
 interface SitePreviewProps {
 	isFSEActive: boolean;
+	showEditSite: boolean;
```

Comment:
You can make these optional by adding a question mark before the colon. I guess maybe having the `= true` in each case might do that automatically, but it's good to be explicit in this case.